src/snapred/backend/recipe/algorithm/BufferMissingColumnsAlgo.py

```diff
@@ -0,0 +1,73 @@
+from mantid.api import AlgorithmFactory, ITableWorkspaceProperty, PropertyMode, PythonAlgorithm
+from mantid.kernel import Direction
+
+from snapred.backend.recipe.algorithm.MantidSnapper import MantidSnapper
+
+
+class BufferMissingColumnsAlgo(PythonAlgorithm):
+    """
+    Record a workspace in a state for the CIS to view later
```

Comment:
I need to still clean this up and write tests for it despite codecov not complaining.